leobot_control/config/gazebo/leobot_control.yml

```diff
@@ -2,6 +2,11 @@ joint_state_controller:
   type: joint_state_controller/JointStateController
   publish_rate: 50
 
+head_position_controller:
+  type: effort_controllers/JointPositionController
+  joint: head_camera_stick_joint
+  pid: {p: 0.0, i: 0.0, d: 0.1}
```

Comment:
@MaxZhaloba why you have started with d parameter? Usually p is set and i and d are tuned in case of "complications". Could you give it a try and tune controller with p gain only